ui_base/src/main/java/ua/gov/diia/ui_base/components/molecule/input/InputGroupMolecule.kt

```diff
@@ -76,7 +76,7 @@ data class InputGroupMoleculeData(val items: List<TextInputMoleculeData>) : Inpu
 @Composable
 @Preview
 fun InputGroupMoleculePreview() {
-    val PHONE_NUMBER_VALIDATION_PATTERN = "^38(039|050|063|066|067|068|073|091|092|093|094|095|096|097|098|099)\\d{7}\$"
+    val PHONE_NUMBER_VALIDATION_PATTERN = "^38(039|050|063|066|067|068|073|075|077|091|092|093|094|095|096|097|098|099)\\d{7}\$"
```

Comment:
isn't it better to use only 1 source of truth instead of keeping this pattern in 6 places? PhoneNumberExt.PHONE_NUMBER_VALIDATION_PATTERN

Reply:
I'll review this later and find out the solution to handle this. 
At first, I decided not to introduce such a significant changes to the code.
But not, I see that more work is needed and this is the better way to handle this.